notebooks/lesson08_group-project.ipynb

```diff
@@ -0,0 +1,949 @@
+{
```

Comment:
<ul><li>The convention I've adopted in the other lectures is to only capitalise the first word in section headers.</li><li>Could you include a "References" section and say this is checklist is taken from Appendix B of the Hands-On ML book by A. Geron?</li><li>May 28 is a Thursday, but I am happy for them to submit on Friday the 29th. We should specify the time, e.g. before 23:59:59 CEST so they don't try to sneak in an extra weekend.</li><li>June 4 is a Thursday</li><li>I would drop the classification reference examples to avoid confusing them (we did it last year because there was a mix)</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>good catch with the dates</p>